tensorflow/python/kernel_tests/variable_scope_test.py

```diff
@@ -680,16 +681,16 @@ def testInitFromNonInitializer(self):
       i = 0
       # Use different varibale_name to distinguish various dtypes
       for dtype in TYPES:
-        x = tf.get_variable(name='x'+str(i), shape=(3,4), dtype=dtype,
-                partitioner=axis0_into2_partitioner(shape=(3,4)))
-        y = tf.get_variable(name='y'+str(i), dtype=dtype, 
-                partitioner=axis0_into2_partitioner(shape=(3,4)),
-                initializer=init_ops.zeros_initializer(shape=(3,4), dtype=dtype))
+        x = tf.get_variable(name='x%d' % i, shape=(3,4), dtype=dtype,
+                partitioner=axis0_into2_partitioner)
+        y = tf.get_variable(name='y%d' % i, dtype=dtype, 
+                partitioner=axis0_into2_partitioner),
+                initializer=init_ops.zeros_initializer(shape=(6,4), dtype=dtype))
```

Comment:
Because using `axis0_into2_partitioner`,  `y` is required a replica shape in 1-D of `x`.